udata/core/dataset/models.py

```diff
@@ -111,13 +111,27 @@ def __unicode__(self):
         return self.title
 
     @classmethod
-    def guess(cls, text):
+    def guess(cls, *strings, **kwargs):
+        '''
+        Try to guess a license from a list of strings.
```

Comment:
Maybe document the `default` kwarg?

Reply:
Or declare it explicitly in the method signature.

Reply:
It's not possible to declare explicitely a keyword argument with variable arguments in Python 2 (already tried :/).
I'll add it to the documentation.